src/services/behaviors/index.ts

```diff
@@ -81,15 +76,16 @@ export const performUpdate = (
     const { parentNode } = targetElement;
     if (parentNode) {
       parentNode.replaceChild(newElement, targetElement);
-      return shallowCloneToRoot((parentNode: any));
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      return shallowCloneToRoot(parentNode as any);
```

Comment:
anyway we can cast as `Document`?

Reply:
Probably change to
`return shallowCloneToRoot(parentNode) as Document;`